bin/validators.js

```diff
@@ -48,6 +48,38 @@ module.exports = {
     return Boolean(firstOption && firstOption.allowIndentationTabs);
   },
 
+  "unicorn/template-indent"({ options }) {
+    if (options.length === 0) {
+      return false;
+    }
+
+    const firstOption = options[0];
+
+    if (!firstOption) {
+      return false;
+    }
+
+    if (!(firstOption.tags || firstOption.comments)) {
+      return true;
+    }
+
+    const tagsAndComments = new Set([
```

Comment:
I don't think we should consider these two types same. They're case sensitive.

Reply:
You’re correct I just checked

Reply:
I’ll get there 😄🤣